packages/@okta/vuepress-site/docs/reference/api/policy/index.md

```diff
@@ -1478,65 +1387,62 @@ The following conditions may be applied to Password Policy
+The following conditions may be applied to the Rules associated with Password Policy:
 
-[Network Condition](#network-condition-object)
+* [People Condition](#people-condition-object)
 
-## IdP Discovery Policy
+* [Network Condition](#network-condition-object)
 
-The IdP Discovery Policy determines where to route users when they are attempting to log in to your org. Users can be routed to a variety of identity providers (`SAML2`, `IWA`, `AgentlessDSSO`, `X509`, `FACEBOOK`, `GOOGLE`, `LINKEDIN`, `MICROSOFT`, `OIDC`) based on multiple conditions listed below.
+## IdP Discovery Policy
 
-For an introduction to the topic, see [IdP Discovery](/docs/concepts/identity-providers/#idp-discovery).
+The IdP Discovery Policy determines where to route Users when they are attempting to sign in to your org. Users can be routed to a variety of identity providers (`SAML2`, `IWA`, `AgentlessDSSO`, `X509`, `FACEBOOK`, `GOOGLE`, `LINKEDIN`, `MICROSOFT`, `OIDC`) based on multiple conditions. For an introduction to the topic, see [IdP Discovery](/docs/concepts/identity-providers/#idp-discovery).
 
-All Okta orgs contain one and only one IdP Discovery Policy, with an immutable default rule routing to your org's login page.
+All Okta orgs contain one and only one IdP Discovery Policy, with an immutable default Rule routing to your org's sign-in page.
 
-### Policy Conditions
-The following conditions may be applied to IdP Discovery Policy
+### Policy conditions
 
-[Network Condition](#network-condition-object)
+The following conditions may be applied to IdP Discovery Policy:
 
-[Platform Condition](#platform-condition-object)
+* [Network Condition](#network-condition-object)
 
-[User Identifier Condition](#user-identifier-condition-object)
+* [Platform Condition](#platform-condition-object)
 
-[Application and App Instance Condition](#application-and-app-instance-condition-object)
+* [User Identifier Condition](#user-identifier-condition-object)
 
-### Policy Action Data
+* [Application and App Instance Condition](#application-and-app-instance-condition-object)
 
+### Policy Action data
 
 #### Policy Action object
 
-
 | Property  | Description                                                          | Data Type | Required |
 | ---       | ---                                                                  | ---       | ---      |
-| providers | List of configured identity providers that a given rule can route to | array     | Yes      |
+| providers | List of configured identity providers that a given Rule can route to | array     | Yes      |
 
-> **Note:** Currently, this `providers` list only supports one value. IdP types `OKTA`, `AgentlessDSSO`, and `IWA` do not require an `id`.
+> **Note:** Currently, this `providers` list only supports one value. IdP types `OKTA`, `AgentlessDSSO`, and `IWA` don't require an `id`.
 
 #### Policy Action Example
 
+> **Note:** Only one value is supported for `providers`.
+
 ```json
   "actions": {
     "idp": {
```

Comment:
Looks like you added a note. Is the intent to remove comment ` //only supports one value`  below. Just double checking.

Reply:
@haishengwu-okta Yes - the intent was to include the comment information in a note rather than in the example code, as you said that we shouldn't have comments in json, correct?